CommonConfig.mjs

```diff
@@ -0,0 +1,447 @@
+        "no-unneeded-ternary": ["error"],
+        "no-unreachable": ["error"],
+        "no-unreachable-loop": ["error"],
+        "no-unsafe-finally": ["error"],
+        "no-unsafe-negation": ["error"],
+        "no-unsafe-optional-chaining": ["error"],
+        "no-unused-expressions": ["error"],
+        "no-unused-labels": ["error"],
+        "no-unused-private-class-members": ["error"],
+        "no-unused-vars": [
+            "warn",
+            {args: "none"},
+        ],
+        "no-use-before-define": ["error"],
+        "no-useless-assignment": ["error"],
+        "no-useless-backreference": ["error"],
+        "no-useless-call": ["error"],
+        "no-useless-catch": ["error"],
+        "no-useless-computed-key": ["error"],
+        "no-useless-concat": ["error"],
+        "no-useless-constructor": ["error"],
+        "no-useless-escape": ["error"],
+        "no-useless-rename": ["error"],
+        "no-useless-return": ["error"],
+        "no-var": ["error"],
+        "no-void": ["error"],
+        "no-warning-comments": [
+            "warn",
+            {
+                location: "start",
+                terms: [
+                    "todo",
+                    "fixme",
+                ],
+            },
+        ],
+        "no-with": ["error"],
+        "object-shorthand": [
+            "error",
+            "consistent",
+        ],
+        "one-var": [
+            "error",
+            {var: "never", let: "never", const: "never"},
+        ],
+        "operator-assignment": ["error"],
+        "prefer-arrow-callback": ["error"],
+        "prefer-const": [
+            "error",
+            {destructuring: "all", ignoreReadBeforeAssign: false},
+        ],
+        "prefer-destructuring": ["error"],
+        "prefer-exponentiation-operator": ["error"],
+        "prefer-named-capture-group": ["off"],
+        "prefer-numeric-literals": ["error"],
+        "prefer-object-has-own": ["error"],
+        "prefer-object-spread": ["error"],
+        "prefer-promise-reject-errors": ["error"],
+        "prefer-regex-literals": ["error"],
+        "prefer-rest-params": ["error"],
+        "prefer-spread": ["error"],
+        "prefer-template": ["error"],
+        "radix": ["error"],
+        "require-atomic-updates": ["warn"],
+        "require-await": ["warn"],
+        "require-unicode-regexp": ["off"],
+        "require-yield": ["error"],
+        "simple-import-sort/exports": ["warn"],
+        "simple-import-sort/imports": [
+            "warn",
+            {
+                groups: [
+                    ["^node:"],
+                    [
+                        "^meteor",
+                        "^prop-types",
+                        "^react",
+                    ],
+                    ["^@?\\w"],
+                    ["^@mui/icons-material"],
+                    ["^/"],
+                    ["^"],
+                    ["^\\."],
+                    ["^\\u0000"],
```

Comment:
Can you write a quick example of imports that would be sorted by each rule?

Reply:
Also removed group `["^"]`